netjsonconfig/backends/openvpn/converters.py

```diff
@@ -46,8 +47,39 @@ def __intermediate_vpn(self, config, remove=[False, 0, '']):
         # do not display status-version if status directive not present
         if 'status' not in config and 'status_version' in config:
             del config['status_version']
+        config = self.__add_tls_auth_key(config)
         return self.sorted_dict(config)
 
+    def __add_tls_auth_key(self, config):
+        tls_auth = config.get('tls_auth', None)
+        if not tls_auth:
+            return config
+        tls_auth = tls_auth.strip()
+        if len(tls_auth.split(' ')) == 1:
+            # The TLS Auth key is present in the field.
+            ca_path = config.get('ca', '')
+            tls_auth_path = '/'.join(ca_path.split('/')[:-1] + ['tls_auth.key'])
```

Comment:
What if there's multiple VPN servers or clients, do the files collide? If yes, please add a test for this and make sure the tls key is unique (eg: each config has a name which can be used for this).